website/versioned_docs/version-0.7.0/configuration/securing-access-to-the-dashboard.mdx

```diff
@@ -11,13 +11,6 @@ There are 2 supported methods for logging in to the dashboard:
 
 The recommended method is to integrate with an OIDC provider, as this will let you control permissions for existing users and groups that have already been configured to use OIDC. However, it is also possible to use a cluster user account to login, if an OIDC provider is not available to use. Both methods work with standard Kubernetes RBAC.
 
-:::note FEATURE TOGGLE
-The following instructions describe a feature that is behind a feature toggle. To enable this feature, set the following OS environment variable:
-```sh
-export WEAVE_GITOPS_AUTH_ENABLED=true
-```
-:::
-
```

Comment:
Let's leave this in there for the 0.7.0 docs, 0.7.0 release probably requires it?